G/GStreamer/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GStreamer"
+version = v"1.18.3"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-$(version).tar.xz", "0c2e09e18f2df69a99b5cb3bd53c597b3cc2e35cf6c98043bb86a66f3d312100")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+apk add bash-completion
+apk add libcap gettext
```

Comment:
Why do you install libcap here if it's a dependency?

Reply:
Without it configure displays:

```
Program setcap found: NO
```

With it, configure displays:

```
Program setcap found: YES (/usr/sbin/setcap)
```

I figure it needed the host version too?

Reply:
> I figure it needed the host version too?

Why?

Reply:
Uhm, I see

Reply:
But if it looks for the host program, I don't understand why it needs the library for the target

Reply:
Dunno. I thought that for binaries called from the command line during compilation like `/usr/sbin/setcap`, you need the host version because you can't really even run target binaries? Or am I still confused?

Reply:
I mean, is `libcap_jll` necessary at this point?

Reply:
There's also a line in configure that says:

```
Run-time dependency libcap found: YES 2.27
```

And I thought that the c libraries were supposed to be compiled for the target?